### DISCUSSION TITLE
Feedback regarding Slash Command output in the chat buffer

### DISCUSSION THREAD

**Discussion by olimorris** [olimorris/codecompanion.nvim#309] [Polls]
I'm finding that putting the output from Slash Commands directly in the buffer looks a little messy:

![2024-10-08 19_17_38 - WezTerm](https://github.com/user-attachments/assets/73e31694-8f8c-4742-a69d-a633e7ce9968)

The folds can open and close which generally makes it feel cluttered. My thoughts are to make this completely invisible and just notify the user that it was added. You could technically see the output by pressing `gd` in the buffer to get the total messages table.

Adding in an option to toggle visibility in the config is an option, although it will add a bit of bloat to the slash commands themselves.

But keen to get the view of you, the users.

**Comment by S1M0N38** [comment]
This is quite neat especially for buffer/file that contains triple backticks which have a folding issue (maybe folding should be base on tree sitter). 

fun fact: even GitHub mobile doesn't let me type triple backticks properly when writing this very comment. 😆

**Comment by olimorris** [comment]
Implemented in `8.3.0`. Thanks for your feedback
